openapi/openapi.yaml

```diff
@@ -185,14 +185,25 @@ paths:
     get:
       summary: Returns system metadata info by id
       operationId: systemsGetById
-      description: Returns a system_metadata by id. id is the DB id so it is mostly used internally.
+      description: "Returns a system_metadata by id. id is the DB id so it is mostly used internally.
+      If with_user_info is set to true and the user has permssion, the returned system_metadata will 
+      contain user information such as shared users."
+      security:
+        - ApiKeyAuth: []
+        - BearerAuth: []
+        - {}
       parameters:
         - in: path
           name: system_id
           schema:
             type: string
           required: true
           example: "619f8ef56e638da17b06b38a"
+        - in: query
```

Comment:
This is not absolutely necessary to fix, but I kinda felt that this flag is not necessary. Couldn't we just return this info all the time if the user has permission, and not return it if the user doesn't have permission?

Reply:
Thanks, and I agree! 

I initially added the flag so shared users are rejected when they click edit (which sends a `with_user_info=true` request) because the button should only allow creators. But a cleaner solution should be disabling the button for all non-creators. I originally planned to submit a different PR for it, but it makes more sense to add it here now along with removing the flag. Working on it now!